### ISSUE TITLE
Execution fails when adding images

### ISSUE THREAD

**Issue by kmfarley11** [Triangle345/GT#24]
If an image is located anywhere other than the directory from which "go run" is called, then the execution fails.

We should probably just make an EngineImages or ExampleImages folder in our Assets directory. So that we do not have to have to make test.png appear in every directory that we create an example in.

The following error will be generated if go run is called from the wrong location:

> Cannot create image: Cannot get valid image section for path: _image we attempted to look for_

This happens because the images needed may not be added to the AggregateImage on init().

See [AggregateImage.go line 23: ](https://github.com/Triangle345/GT/blob/master/Graphics/Image/ImageAggretator.go#L23)
the init function runs LoadImages("./")
This looks wherever go run is called from. But the aggregate image should look in our Engine's Images folder and in our project's generated resource folder (once project creation / pathing gets worked out as in #13).

**Comment by Triangle345** [comment]
I suggest placing all images in the "Assets" folder where we currently have fonts. Just create another folder called images or something

**Comment by kmfarley11** [comment]
As per the suggestion, we now use GT/Assets/Images for engine image assets. If it looks good enough we can close this issue and move on. True relative pathing will have to be addressed from #13, and when that happens we may want to consider adding configurations for easy engine asset reference. But for now the user will need to add an image to the folder and reference its absolute path in their code.

**Comment by Triangle345** [comment]
```
Font.LoadFonts("../Assets/Fonts")
imgsPath, err := filepath.Abs("../Assets/Images")
```

still uses relative pathing though. Do we want to maybe do this through gopath variable as we are sure it should be set up

**Comment by kmfarley11** [comment]
Fonts still work fine according to the above commit. I updated and re-ran all tests to verify its all good. Relative pathing works from the working directory, and we use it all over the code. To do true relative pathing we'd either have to use the GOPATH or set up a config file which defines asset locations. This subject merits some debate and should be discussed in person if we can. 

As for this issue though, the bug here was fixed in that we can now put an image in a defined location (do not have to have test.png in the same folder as where go run is called from). So I suggest this issue be closed, and the discussion / implementation for relative pathing continue in either #13 or a new issue.
